<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-QMAKQJB" failures="3" tests="20" name="com.vinGuru.tests.flightResevearion.FlightReseverationTest" time="41.842" errors="0" timestamp="2024-07-23T14:25:23 IST" skipped="0">
  <testcase classname="com.vinGuru.tests.flightResevearion.FlightReseverationTest" name="userRegistaraionTest" time="2.743"/>
  <system-out/>
  <testcase classname="com.vinGuru.tests.flightResevearion.FlightReseverationTest" name="userRegistaraionTest" time="2.357"/>
  <system-out/>
  <testcase classname="com.vinGuru.tests.flightResevearion.FlightReseverationTest" name="registartionConfirmationPage" time="2.539"/>
  <system-out/>
  <testcase classname="com.vinGuru.tests.flightResevearion.FlightReseverationTest" name="registartionConfirmationPage" time="1.812"/>
  <system-out/>
  <testcase classname="com.vinGuru.tests.flightResevearion.FlightReseverationTest" name="flightReseverationConfirmationTest" time="0.888">
    <failure message="expected [$1753 USD] but found [$584 USD]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [$1753 USD] but found [$584 USD]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.vinGuru.tests.flightResevearion.FlightReseverationTest.flightReseverationConfirmationTest(FlightReseverationTest.java:80)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
]]>
    </failure>
  </testcase> <!-- flightReseverationConfirmationTest -->
  <system-out/>
  <testcase classname="com.vinGuru.tests.flightResevearion.FlightReseverationTest" name="flightSelectionTest" time="1.916"/>
  <system-out/>
  <testcase classname="com.vinGuru.tests.flightResevearion.FlightReseverationTest" name="flightSelectionTest" time="2.142"/>
  <system-out/>
  <testcase classname="com.vinGuru.tests.flightResevearion.FlightReseverationTest" name="userRegistaraionTest" time="1.477"/>
  <system-out/>
  <testcase classname="com.vinGuru.tests.flightResevearion.FlightReseverationTest" name="flightSelectionTest" time="2.132"/>
  <system-out/>
  <testcase classname="com.vinGuru.tests.flightResevearion.FlightReseverationTest" name="userRegistaraionTest" time="2.361"/>
  <system-out/>
  <testcase classname="com.vinGuru.tests.flightResevearion.FlightReseverationTest" name="flightSelectionTest" time="2.282"/>
  <system-out/>
  <testcase classname="com.vinGuru.tests.flightResevearion.FlightReseverationTest" name="flightReseverationConfirmationTest" time="2.514">
    <failure message="expected [$1169 USD] but found [$584 USD]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [$1169 USD] but found [$584 USD]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.vinGuru.tests.flightResevearion.FlightReseverationTest.flightReseverationConfirmationTest(FlightReseverationTest.java:80)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
]]>
    </failure>
  </testcase> <!-- flightReseverationConfirmationTest -->
  <system-out/>
  <testcase classname="com.vinGuru.tests.flightResevearion.FlightReseverationTest" name="registartionConfirmationPage" time="0.748"/>
  <system-out/>
  <testcase classname="com.vinGuru.tests.flightResevearion.FlightReseverationTest" name="flightReseverationConfirmationTest" time="1.819">
    <failure message="expected [$2338 USD] but found [$584 USD]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [$2338 USD] but found [$584 USD]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.vinGuru.tests.flightResevearion.FlightReseverationTest.flightReseverationConfirmationTest(FlightReseverationTest.java:80)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
]]>
    </failure>
  </testcase> <!-- flightReseverationConfirmationTest -->
  <system-out/>
  <testcase classname="com.vinGuru.tests.flightResevearion.FlightReseverationTest" name="flightReseverationConfirmationTest" time="2.141"/>
  <system-out/>
  <testcase classname="com.vinGuru.tests.flightResevearion.FlightReseverationTest" name="flightSearchTest" time="2.212"/>
  <system-out/>
  <testcase classname="com.vinGuru.tests.flightResevearion.FlightReseverationTest" name="registartionConfirmationPage" time="2.455"/>
  <system-out/>
  <testcase classname="com.vinGuru.tests.flightResevearion.FlightReseverationTest" name="flightSearchTest" time="2.442"/>
  <system-out/>
  <testcase classname="com.vinGuru.tests.flightResevearion.FlightReseverationTest" name="flightSearchTest" time="2.472"/>
  <system-out/>
  <testcase classname="com.vinGuru.tests.flightResevearion.FlightReseverationTest" name="flightSearchTest" time="2.390"/>
  <system-out/>
</testsuite> <!-- com.vinGuru.tests.flightResevearion.FlightReseverationTest -->
